helix-core/src/test/java/org/apache/helix/integration/task/TaskTestUtil.java

```diff
@@ -231,9 +239,17 @@ public static Date getDateFromStartTime(String startTime)
 
   public static JobQueue.Builder buildJobQueue(String jobQueueName, int delayStart,
       int failureThreshold, int capacity) {
+    return buildJobQueue(jobQueueName, delayStart, failureThreshold, capacity, false);
+  }
+
+  public static JobQueue.Builder buildJobQueue(String jobQueueName, int delayStart,
+      int failureThreshold, int capacity, boolean disablePurge) {
```

Comment:
nit, but it would be more intuitive to be "boolean doPurge"

Reply:
Changed to do purge